openwisp_users/apps.py

```diff
@@ -71,3 +82,29 @@ def update_organizations_dict(cls, instance, **kwargs):
         cache.delete(cache_key)
         # forces caching
         user.organizations_dict
+
+    def create_organization_owner(cls, instance, **kwargs):
+        if not kwargs.get('created') or not instance.is_admin:
+            return
+        owner_model = load_model('openwisp_users', 'OrganizationOwner')
+        model = kwargs.get('sender')
+        org_users = model.objects.filter(organization=instance.organization)
+        is_first = org_users.first() == org_users.last() == instance
+        if (
+            is_first
+            and not owner_model.objects.filter(organization_user=instance).exists()
+        ):
+            with transaction.atomic():
+                try:
+                    owner = owner_model(
+                        organization_user=instance, organization=instance.organization
+                    )
+                    owner.full_clean()
+                    owner.save()
+                except ValidationError as e:
+                    logger.exception(
+                        'Got exception {} while saving OrganizationOwner with '
+                        'organization_user {} and organization {}'.format(
+                            type(e), instance, instance.organization
+                        )
```

Comment:
we can use use f strings instead of `format`, since we only support python >= 3.6 now

Reply:
Ok